ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```
@@ -870,6 +870,42 @@ presubmits:
             cpu: 10m
       serviceAccountName: ci-operator
     trigger: '(?m)^/test (?:.*? )?unit(?: .*?)?$'
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/verify-vendor
+    decorate: true
+    decoration_config:
+      grace_period: 0s
+      skip_cloning: true
+      timeout: 0s
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+    name: pull-ci-openshift-installer-master-verify-vendor
+    rerun_command: /test verify-vendor
+    spec:
```

```suggestion
    run_if_changed: (Gopkg.toml|Gopkg.lock)
    spec:
```
We could do something like this in combination with always  `always_run: false` to only run this test if there are changes to these files. I'm a vendoring newb, so I'm not sure if that trigger would catch all vendoring changes, or if we need other checks as well, such as changes in the vendor folder.

---

This test will be much faster than our e2e clusters, so I'm not concerned about running it every time.

---

This was generated by prow-gen so I didn't want to make any changes.

Personally verify scripts like others already run always, and these are quick enough to not require special logic.

And I don't want to remember to update the regex when we move to go mod, _i know I will forget_ ;)